packages/react-admin/src/index.ts

```diff
@@ -5,5 +5,11 @@ import AdminRouter from './AdminRouter';
 import defaultI18nProvider from './defaultI18nProvider';
 
 export * from 'ra-core';
+/* 
+error - what do to? which one is the correct symbol?
+Module 'ra-core' has already exported a member named 'Notification'. Consider explicitly re-exporting to resolve the ambiguity.ts(2308)
+Module 'ra-core' has already exported a member named 'Pagination'. Consider explicitly re-exporting to resolve the ambiguity.ts(2308)
+*/
+// @ts-ignore
```

Comment:
@fzaninotto what's the correct symbols here should be?

Reply:
I believe ra-ui-material-ui should import those types from ra-core